.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+- Explain why this enhancement would be useful to most TypicalBot users.
+
+### Pull Requests
+
+In order to have your pull request considered for review, it must meet the following requirements:
+
+- A completed [pull request template](https://github.com/typicalbot/typicalbot/blob/master/.github/PULL_REQUEST_TEMPLATE.md). If the pull request template is not completed, you'll be asked to complete it before your pull request will be considered for merging.
+- Compliance with the [style guides](#style-guides). If your pull request doesn't comply with one or more of the style guides, you'll be asked to bring it to complaince before your pull request will be considered for merging.
+- Completed documentation and tests, if applicable.
+
+> If your pull request closes an issue, state so under the Issues section in your pull request like so: `Closes #<issue>`. This helps ensure that GitHub automatically closes the issue when your pull request is merged.
+
+> The reviewer(s) of your pull request may request changes or ask you to complete additional tasks, tests, or other changes before your pull request will be accepted and merged.
+
+## Style Guides
+
+### CHANGELOG
+
+Use [Keepachangelog.org](https://keepachangelog.com/en/1.0.0/) as a guide for adding changes to the CHANGELOG.
+
+Each item should be in the following format: `<package>: <change> (#<pr>) (<commit>)[, closes #<issue>]`.
+
+- Replace `<package>` with the name of the package the change applies to.
+- Replace `<change>` with a description of the change (usually the commit name will suffice).
+- Replace `<pr>` with the pull request number where the change was proposed.
+- Replace `<commit>` with the hash of the commit where the change was made.
+- Replace `<issue>` with the issue that the change closes, if applicable.
+
+### Git Commit Messages
+
+- Follow [Conventional Commits v1.0.0-beta.4](https://www.conventionalcommits.org/en/v1.0.0-beta.4/). Allowed commit types are: `chore:`, `ci:`, `docs:`, `enhancement:`, `feat:` `fix:`, `pref:`, `style:`, `refactor:`, and `test:`. Commit types should be in lowercase. If a commit fits into multiple commit types, go back and make multiple commits whenever possible.
+- Use the imperative mood (i.e. "Move command to..." not "Moves command to...").
+- Use the present tense (i.e. "Add feature" not "Added feature").
+- Keep commits discrete. Avoid including multiple unrelated changes in a single commit.
+- Keep commits self-contained. Avoid spreading a single change across multiple commits.
+- Reference issues and pull requests liberally after the first line.
+
+### JavaScript Style Guide
+
+All JavaScript code must adhere to the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript), with the exception of [indents (19.1)](https://github.com/airbnb/javascript#whitespace--spaces). Use soft tabs (space character) set to 4 spaces.
+
+### Documentation
+
+All code must be documented using [ESDoc tags](https://esdoc.org/manual/tags.html).
```

Comment:
Remove this section - I removed esdoc from the current version and I don’t know if I want to use it for the new version.

Reply:
Are we not going to replace it with something?